README.md

```diff
@@ -1,73 +1,83 @@
 # Tarantool
 
-[![Build Status][travis-badge]][travis-url]
-[![Build Status][gitlab-ci-badge]][gitlab-ci-url]
+[![Actions Status][actions-badge]][actions-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
+[Tarantool][tarantool-url] is an in-memory database and application server.
 
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
-
-Tarantool is an in-memory database and application server.
+It is distributed under [BSD 2-Clause][license] terms.
 
 Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+
+* 100% compatible drop-in replacement for Lua 5.1, based on LuaJIT 2.1.
+  Simply use `#!/usr/bin/tarantool` instead of `#!/usr/bin/lua` in your script.
+* Full support for Lua modules and a rich set of [own modules][modules],
```

Comment:
I would repeat concerns on compatibility. I believe trying to run arbitrary Lua modules can cause real problems due to the cooperative multitasking mentioned in the same sentence.

Reply:
Okay. Changed this way:

```markdown
Key features of the application server:

* Heavily optimized Lua interpreter with incredibly fast tracing JIT compiler,
  based on LuaJIT 2.1.
* Cooperative multitasking, non-blocking IO.
* [Persistent queues][queue].
* [Sharding][vshard].
* [Cluster and application management framework][cartridge].
* Access to external databases such as [MySQL][mysql] and [PostgreSQL][pg].
* And many other built-in and standalone [modules][modules].
```